src/main/java/com/jwplayer/southpaw/Southpaw.java

```diff
@@ -515,13 +520,15 @@ protected void createDenormalizedRecords(
                         dePrimaryKey.getBytes(),
                         newDeRecord
                 );
+                nbWrittenDenormalizedRecords += 1;
             }
             metrics.denormalizedRecordsCreated.mark(1);
             metrics.denormalizedRecordsCreatedByTopic.get(root.getDenormalizedName()).mark(1);
             metrics.denormalizedRecordsToCreate.update(metrics.denormalizedRecordsToCreate.getValue() - 1);
             metrics.denormalizedRecordsToCreateByTopic.get(root.getDenormalizedName())
                     .update(metrics.denormalizedRecordsToCreateByTopic.get(root.getDenormalizedName()).getValue() - 1);
         }
+        logger.info(String.format("Ordered the creation of %d %s denormalized records", nbWrittenDenormalizedRecords, root.getEntity()));
```

Comment:
rather than logging should this be a metric that you `mark` with it's current value?

Reply:
Was mostly for debugging, instead of reading downstream topics